rules/windows/process_creation/win_visual_basic_compiler.yml

```diff
@@ -0,0 +1,22 @@
+title: Visual Basic Command Line Compiler Usage
+id: 7b10f171-7f04-47c7-9fa2-5be43c76e535
+status: experimental
+description: Detects successful code compilation via Visual Basic Command Line Compiler that utilizes Windows Resource to Object Converter.
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Vbc/
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community' 
```

Comment:
Hello @esebese !
Could you please apply [this](https://github.com/Neo23x0/sigma/pull/1062#discussion_r502062289) recommendation here as well?